azurerm/internal/services/datafactory/data_factory_trigger_tumbling_window_resource_test.go

```diff
@@ -0,0 +1,300 @@
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDataFactoryTriggerDestroy("azurerm_data_factory_trigger_tumbling_window"),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDataFactoryTriggerTumblingWindow_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataFactoryTriggerExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMDataFactoryTriggerTumblingWindow_startstop(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_data_factory_trigger_tumbling_window", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDataFactoryTriggerDestroy("azurerm_data_factory_trigger_tumbling_window"),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDataFactoryTriggerTumblingWindow_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataFactoryTriggerExists(data.ResourceName),
+					testCheckAzureRMDataFactoryTriggerStarts(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+			{
+				Config:             testAccAzureRMDataFactoryTriggerTumblingWindow_basic(data),
+				ExpectNonEmptyPlan: false,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataFactoryTriggerStops(data.ResourceName),
+				),
+			},
+			{
+				Config:             testAccAzureRMDataFactoryTriggerTumblingWindow_basic(data),
+				ExpectNonEmptyPlan: false,
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMDataFactoryTriggerTumblingWindow_trigger_dependency(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_data_factory_trigger_tumbling_window", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDataFactoryTriggerDestroy("azurerm_data_factory_trigger_tumbling_window"),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDataFactoryTriggerTumblingWindow_trigger_dependency(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataFactoryTriggerExists(data.ResourceName),
+					testCheckAzureRMDataFactoryTriggerStarts(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+			{
+				Config:             testAccAzureRMDataFactoryTriggerTumblingWindow_trigger_dependency(data),
+				ExpectNonEmptyPlan: false,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataFactoryTriggerStops(data.ResourceName),
+				),
+			},
+			{
+				Config:             testAccAzureRMDataFactoryTriggerTumblingWindow_trigger_dependency(data),
+				ExpectNonEmptyPlan: false,
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMDataFactoryTriggerTumblingWindow_complete(t *testing.T) {
```

Comment:
as this updates it should be
```suggestion
func TestAccAzureRMDataFactoryTriggerTumblingWindow_update(t *testing.T) {
```